examples/ssd1306_example.py

```diff
@@ -0,0 +1,15 @@
+# pylint: skip-file
+import ssd1306
+from machine import Pin, SoftI2C
+
+i2c = SoftI2C(scl=Pin(5), sda=Pin(4))
+
+oled_width = 128
+oled_height = 64
+oled = ssd1306.SSD1306_I2C(oled_width, oled_height, i2c)
```

Comment:
pylint heeft gelijk, de class heet SSD1306I2C

Reply:
Nee, achteraf heette die SSD1306SPI

Reply:
hmm, dan is de oled wel via SPI aangesloten en niet via I2C. Het standaard leaphy oledje is I2C, dus dan is het voorbeeld wat vreemd.